From 50846ce1fe94e0d51e427e3e5143ce50ef0c14d2 Mon Sep 17 00:00:00 2001
From: Michael W <baddaemon87@gmail.com>
Date: Thu, 25 Jan 2018 19:27:40 +0100
Subject: [PATCH 087/249] wiki: Winter cleanup

* Remove not required files (including "posts")
* Remove not required logic (disqus, feedback, github editme,
  google analytics, ...)
* Make use of liquid 4 syntax: Remove some empty lines which exist because
  the liquid statements are executed rather than rendered, but their
  occupied space is not freed without the newer syntax (adding - before
  and/or after the statement)

Change-Id: I2085814b24480744d4088b5387661e6347242824
---
 _config.yml                                   |  20 ---
 _includes/archive.html                        |  15 --
 _includes/callout.html                        |   1 -
 _includes/custom/getting_started_series.html  |  19 ---
 .../custom/getting_started_series_next.html   |  10 --
 _includes/custom/series_acme.html             |  19 ---
 _includes/custom/series_acme_next.html        |  30 ----
 _includes/custom/sidebarconfigs.html          |  33 -----
 _includes/custom/usermap.html                 |  14 --
 _includes/custom/usermapcomplex.html          |  91 ------------
 _includes/disqus.html                         |  16 ---
 _includes/feedback.html                       |  13 --
 _includes/google_analytics.html               |   6 -
 _includes/initialize_shuffle.html             | 130 ------------------
 _includes/inline_image.html                   |   1 -
 _includes/taglogic.html                       |  32 -----
 _includes/topnav.html                         |  23 ++--
 _layouts/default.html                         |  16 ---
 _layouts/default_deviceinfo.html              |  15 --
 _layouts/deviceinfo.html                      |  26 ----
 _layouts/page.html                            |  58 ++------
 _layouts/post.html                            |   4 -
 search.json                                   |  34 ++---
 sitemap.xml                                   |  23 +---
 24 files changed, 43 insertions(+), 606 deletions(-)
 delete mode 100644 _includes/archive.html
 delete mode 100644 _includes/callout.html
 delete mode 100644 _includes/custom/getting_started_series.html
 delete mode 100644 _includes/custom/getting_started_series_next.html
 delete mode 100644 _includes/custom/series_acme.html
 delete mode 100644 _includes/custom/series_acme_next.html
 delete mode 100644 _includes/custom/sidebarconfigs.html
 delete mode 100644 _includes/custom/usermap.html
 delete mode 100644 _includes/custom/usermapcomplex.html
 delete mode 100644 _includes/disqus.html
 delete mode 100644 _includes/feedback.html
 delete mode 100644 _includes/google_analytics.html
 delete mode 100644 _includes/initialize_shuffle.html
 delete mode 100644 _includes/inline_image.html
 delete mode 100644 _includes/taglogic.html

diff --git a/_config.yml b/_config.yml
index a3645be..1314319 100644
--- a/_config.yml
+++ b/_config.yml
@@ -11,12 +11,6 @@ title: LineageOS Wiki
 company_name: The LineageOS Project
 # this appears in the footer
 
-github_editme_path:
-# if you're using Github, provide the basepath to the branch you've created for reviews, following the sample here. if not, leave this value blank.
-
-disqus_shortname: idrbwjekyll
-# if you're using disqus for comments, add the shortname here. if not, leave this value blank.
-
 host: 127.0.0.1
 # the preview server used. Leave as is.
 
@@ -30,20 +24,6 @@ exclude:
   - scripts
 # these are the files and directories that jekyll will exclude from the build
 
-feedback_subject_line: Jekyll documentation theme
-
-feedback_email: me@example.com
-# used as a contact email for the Feedback link in the top navigation bar
-
-feedback_disable: true
-# if you uncomment the previous line, the Feedback link gets removed
-
-# feedback_text: "Need help?"
-# if you uncomment the previous line, it changes the Feedback text
-
-# feedback_link: "http://helpy.io/"
-# if you uncomment the previous line, it changes where the feedback link points to
-
 highlighter: rouge
 # library used for syntax highlighting
 
diff --git a/_includes/archive.html b/_includes/archive.html
deleted file mode 100644
index 275850c..0000000
--- a/_includes/archive.html
+++ /dev/null
@@ -1,15 +0,0 @@
----
-layout: default
-type: archive
----
-
-<div class="post-header">
-  <h1 class="post-title-main">{{ page.title }}</h1>
-</div>
-<div class="post-content">
-
-{{ content }}
-</div>
-
- 
-
diff --git a/_includes/callout.html b/_includes/callout.html
deleted file mode 100644
index d492b18..0000000
--- a/_includes/callout.html
+++ /dev/null
@@ -1 +0,0 @@
-<div markdown="span" class="bs-callout bs-callout-{{include.type}}">{{include.content}}</div>
diff --git a/_includes/custom/getting_started_series.html b/_includes/custom/getting_started_series.html
deleted file mode 100644
index 5b573a9..0000000
--- a/_includes/custom/getting_started_series.html
+++ /dev/null
@@ -1,19 +0,0 @@
-<div class="seriesContext">
-    <div class="btn-group">
-        <button type="button" data-toggle="dropdown" class="btn btn-primary dropdown-toggle">Getting Started <span class="caret"></span></button>
-        <ol class="dropdown-menu">
-            {% assign pages = site.pages | sort:"weight"  %}
-            {% for p in pages %}
-            {% if p.series == "Getting Started" %}
-            {% if p.url == page.url %}
-            <li class="active"> → {{p.title}}</li>
-            {% else %}
-            <li>
-                <a href="{{p.url | remove: "/"}}">{{p.title}}</a>
-            </li>
-            {% endif %}
-            {% endif %}
-            {% endfor %}
-        </ol>
-    </div>
-</div>
\ No newline at end of file
diff --git a/_includes/custom/getting_started_series_next.html b/_includes/custom/getting_started_series_next.html
deleted file mode 100644
index c464214..0000000
--- a/_includes/custom/getting_started_series_next.html
+++ /dev/null
@@ -1,10 +0,0 @@
-<p>{% assign series_pages = site.tags.series_acme %}
-    {% for p in pages %}
-    {% if p.series == "Getting Started" %}
-    {% assign nextTopic = page.weight | plus: "1"  %}
-    {% if p.weight == nextTopic  %}
-    <a href="{{p.url | remove: "/"}}"><button type="button" class="btn btn-primary">Next: {{p.title}}</button></a>
-    {% endif %}
-    {% endif %}
-    {% endfor %}
-</p>
\ No newline at end of file
diff --git a/_includes/custom/series_acme.html b/_includes/custom/series_acme.html
deleted file mode 100644
index b9ebf44..0000000
--- a/_includes/custom/series_acme.html
+++ /dev/null
@@ -1,19 +0,0 @@
-<div class="seriesContext">
-    <div class="btn-group">
-        <button type="button" data-toggle="dropdown" class="btn btn-primary dropdown-toggle">Series Demo <span class="caret"></span></button>
-        <ol class="dropdown-menu">
-            {% assign pages = site.pages | sort:"weight"  %}
-            {% for p in pages %}
-            {% if p.series == "ACME series" %}
-            {% if p.url == page.url %}
-            <li class="active"> → {{p.weight}}. {{p.title}}</li>
-            {% else %}
-            <li>
-                <a href="{{p.url | remove: "/"}}">{{p.weight}}. {{p.title}}</a>
-            </li>
-            {% endif %}
-            {% endif %}
-            {% endfor %}
-        </ol>
-    </div>
-</div>
\ No newline at end of file
diff --git a/_includes/custom/series_acme_next.html b/_includes/custom/series_acme_next.html
deleted file mode 100644
index 9c9a11a..0000000
--- a/_includes/custom/series_acme_next.html
+++ /dev/null
@@ -1,30 +0,0 @@
-<div class="seriesContext">
-    <a>
-        {% assign pages = site.pages | sort:"weight"  %}
-        {% for pg in pages %}
-        {% if pg.series == "ACME series" %}
-        {% if pg.weight > page.weight %}
-        <a href="{{pg.url | remove: '/'}}"><button type="button" class="btn btn-primary">Next: {{pg.title}}</button></a>
-        {% break %}
-        {% endif %}
-        {% endif %}
-        {% endfor %}
-    </a>
-    &nbsp;
-    <div class="btn-group">
-        <button type="button" data-toggle="dropdown" class="btn btn-primary dropdown-toggle">Jump to: <span class="caret"></span></button>
-        <ol class="dropdown-menu">
-            {% for pg in pages %}
-            {% if pg.series == "ACME series" %}
-            {% if pg.url == page.url %}
-            <li class="active"> → {{pg.weight}}. {{pg.title}}</li>
-            {% else %}
-            <li>
-                <a href="{{pg.url | remove: '/'}}">{{pg.title}}</a>
-            </li>
-            {% endif %}
-            {% endif %}
-            {% endfor %}
-        </ol>
-    </div>
-</div>
\ No newline at end of file
diff --git a/_includes/custom/sidebarconfigs.html b/_includes/custom/sidebarconfigs.html
deleted file mode 100644
index 36167ad..0000000
--- a/_includes/custom/sidebarconfigs.html
+++ /dev/null
@@ -1,33 +0,0 @@
-{% if page.sidebar == "home_sidebar" %}
-{% assign sidebar = site.data.sidebars.home_sidebar.entries %}
-
-{% elsif page.sidebar == "product1_sidebar" %}
-{% assign sidebar = site.data.sidebars.product1_sidebar.entries %}
-
-{% elsif page.sidebar == "product2_sidebar" %}
-{% assign sidebar = site.data.sidebars.product2_sidebar.entries %}
-
-{% elsif page.sidebar == "mydoc_sidebar" %}
-{% assign sidebar = site.data.sidebars.mydoc_sidebar.entries %}
-
-{% else %}
-{% assign sidebar = site.data.sidebars.home_sidebar.entries %}
-{% endif %}
-
-
-
-{% comment %}
-sidebar configuration for print files
-{% endcomment %}
-
-{% if site.product == "mydoc" %}
-{% assign sidebar_pdf = site.data.sidebars.mydoc_sidebar.entries %}
-{% endif %}
-
-{% if site.product == "product1" %}
-{% assign sidebar_pdf = site.data.sidebars.product1_sidebar.entries %}
-{% endif %}
-
-{% if site.product == "product2" %}
-{% assign sidebar_pdf = site.data.sidebars.product2_sidebar.entries %}
-{% endif %}
diff --git a/_includes/custom/usermap.html b/_includes/custom/usermap.html
deleted file mode 100644
index f90b5c2..0000000
--- a/_includes/custom/usermap.html
+++ /dev/null
@@ -1,14 +0,0 @@
-
-<div id="userMap">
-<div class="content"><a href="p2_sample1.html"><div class="box box1">Connect to ADB</div></a></div>
-<div class="arrow">→</div>
-<div class="content"><a href="p2_sample2.html"><div class="box box2">Download and Build the Starter Kit</div></a></div>
-<div class="arrow">→</div>
-<div class="content"><a href="p2_sample3.html"><div class="box box3">Take a Tour</div></a></div>
-<div class="arrow">→</div>
-<div class="content"><a href="p2_sample4.html"><div class="box box4">Load Your Widgets</div></a></div>
-<div class="arrow">→</div>
-<div class="content"><a href="p2_sample5.html"><div class="box box5">Query for Something</div></a></div>
-<div class="clearfix"></div>
-
-</div>
diff --git a/_includes/custom/usermapcomplex.html b/_includes/custom/usermapcomplex.html
deleted file mode 100644
index c75a652..0000000
--- a/_includes/custom/usermapcomplex.html
+++ /dev/null
@@ -1,91 +0,0 @@
-<div id="userMap">
-
-    <!-- Button trigger modal -->
-<button type="button" class="btn btn-default btn-lg modalButton1" data-toggle="modal" data-target="#myModal1">Get Started</button>
-
-    <!-- Modal -->
-    <div class="modal fade" id="myModal1" tabindex="-1" role="dialog" aria-labelledby="myModalLabel">
-      <div class="modal-dialog" role="document">
-        <div class="modal-content">
-          <div class="modal-header">
-            <button type="button" class="close" data-dismiss="modal" aria-label="Close"><span aria-hidden="true">&times;</span></button>
-            <h4 class="modal-title" id="myModalLabel">Get Started</h4>
-          </div>
-          <div class="modal-body">
-<p>This is just dummy text ... Your first steps should be to get started. You will need to do the following:</p>
-
-    <ul>
-        <li><a href="p2_sample6.html">Sample 6</a></li>
-        <li><a href="p2_sample7.html">Sample 7</a></li>
-        <li><a href="p2_sample8.html">Sample 8</a></li>
-    </ul>
-    <p>If you run into any of these setup issues, you must solve them before you can continue on.</p>
-
-          </div>
-          <div class="modal-footer">
-            <button type="button" class="btn btn-default" data-dismiss="modal">Close</button>
-          </div>
-        </div>
-      </div>
-    </div>
-            <span class="complexArrow">→</span>
-
-            <!-- Button trigger modal -->
-            <button type="button" class="btn btn-default btn-lg modalButton2" data-toggle="modal" data-target="#myModal2">Build your widgets</button>
-            <!-- Modal -->
-            <div class="modal fade" id="myModal2" tabindex="-1" role="dialog" aria-labelledby="myModalLabel">
-              <div class="modal-dialog" role="document">
-                <div class="modal-content">
-                  <div class="modal-header">
-                    <button type="button" class="close" data-dismiss="modal" aria-label="Close"><span aria-hidden="true">&times;</span></button>
-                    <h4 class="modal-title" id="myModalLabel">Build your widgets</h4>
-                  </div>
-                  <div class="modal-body">
-                    <p>In this step, you will build the widgets for your system. The widgets form the various components that blah blah blah this is dummy text power the nuclear capabilities of your energy transformer into deep space using wormhole technology and warp drive speeds.</p>
-
-                    <p>In order to configure your widgets, you will need to follow these topics:</p>
-                    <ul>
-                        <li><a href="p2_sample9.html">Sample 9</a></li>
-                        <li><a href="p2_sample10.html">Sample 10</a></li>
-                        <li><a href="p2_sample11.html">Sample 11</a></li>
-                    </ul>
-                  </div>
-                  <div class="modal-footer">
-                    <button type="button" class="btn btn-default" data-dismiss="modal">Close</button>
-                  </div>
-                </div>
-              </div>
-            </div>
-
-            <span class="complexArrow">→</span>
-
-            <!-- Button trigger modal -->
-<button type="button" class="btn btn-default btn-lg modalButton3" data-toggle="modal" data-target="#myModal3">Publish your app</button>
-            <!-- Modal -->
-            <div class="modal fade" id="myModal3" tabindex="-1" role="dialog" aria-labelledby="myModalLabel">
-              <div class="modal-dialog" role="document">
-                <div class="modal-content">
-                  <div class="modal-header">
-                    <button type="button" class="close" data-dismiss="modal" aria-label="Close"><span aria-hidden="true">&times;</span></button>
-                    <h4 class="modal-title" id="myModalLabel">Publish your app</h4>
-                  </div>
-                  <div class="modal-body">
-                    <p>After you've configured all the necessary components to build your space transformer, you need to publish your app. Of course this content is also just dummy text. Pay no particular attention to the content but rather the format and placement of the map.</p>
-                    <p>To publish your app, see the following:
-                        <ul>
-                            <li><a href="p2_sample12.html">Sample 12</a></li>
-                            <li><a href="p2_sample13.html">Sample 13</a></li>
-                            <li><a href="p2_sample14.html">Sample 14</a></li>
-                        </ul>
-                  </div>
-                  <div class="modal-footer">
-                    <button type="button" class="btn btn-default" data-dismiss="modal">Close</button>
-                  </div>
-                </div>
-              </div>
-            </div>
-
-
-<div class="clearfix"></div>
-
-</div>
diff --git a/_includes/disqus.html b/_includes/disqus.html
deleted file mode 100644
index 4fad7a2..0000000
--- a/_includes/disqus.html
+++ /dev/null
@@ -1,16 +0,0 @@
-{% if site.disqus_shortname %}
-
-    <div id="disqus_thread"></div>
-    <script type="text/javascript">
-        /* * * CONFIGURATION VARIABLES: EDIT BEFORE PASTING INTO YOUR WEBPAGE * * */
-        var disqus_shortname = '{{site.disqus_shortname}}'; // required: replace example with your forum shortname
-
-        /* * * DON'T EDIT BELOW THIS LINE * * */
-        (function() {
-            var dsq = document.createElement('script'); dsq.type = 'text/javascript'; dsq.async = true;
-            dsq.src = '//' + disqus_shortname + '.disqus.com/embed.js';
-            (document.getElementsByTagName('head')[0] || document.getElementsByTagName('body')[0]).appendChild(dsq);
-        })();
-    </script>
-    <noscript>Please enable JavaScript to view the <a href="https://disqus.com/?ref_noscript">comments powered by Disqus.</a></noscript>
-{% endif %}
\ No newline at end of file
diff --git a/_includes/feedback.html b/_includes/feedback.html
deleted file mode 100644
index 4b55e44..0000000
--- a/_includes/feedback.html
+++ /dev/null
@@ -1,13 +0,0 @@
-<li>
-{% if site.feedback_text %}
-  {% assign feedback_text = site.feedback_text %}
-{% else %}
-  {% assign feedback_text = "Feedback" %}
-{% endif %}
-
-{% if site.feedback_link %}
-  <a class="email" title="Submit feedback" href="{{site.feedback_link}}">{{feedback_text}}</a>
-{% else %}
-  <a class="email" title="Submit feedback" href="#" onclick="javascript:window.location='mailto:{{site.feedback_email}}?subject={{site.feedback_subject_line}} feedback&body=I have some feedback about the {{page.title}} page: ' + window.location.href;"><i class="fa fa-envelope-o"></i> {{feedback_text}}</a>
-{% endif %}
-<li>
diff --git a/_includes/google_analytics.html b/_includes/google_analytics.html
deleted file mode 100644
index 56b2ee8..0000000
--- a/_includes/google_analytics.html
+++ /dev/null
@@ -1,6 +0,0 @@
-<!-- the google_analytics_id gets auto inserted from the config file -->
-
-{% if site.google_analytics %}
-
-<script>(function(i,s,o,g,r,a,m){i['GoogleAnalyticsObject']=r;i[r]=i[r]||function(){(i[r].q=i[r].q||[]).push(arguments)},i[r].l=1*new Date();a=s.createElement(o),m=s.getElementsByTagName(o)[0];a.async=1;a.src=g;m.parentNode.insertBefore(a,m)})(window,document,'script','//www.google-analytics.com/analytics.js','ga');ga('create','{{site.google_analytics}}','auto');ga('require','displayfeatures');ga('send','pageview');</script>
-{% endif %}
\ No newline at end of file
diff --git a/_includes/initialize_shuffle.html b/_includes/initialize_shuffle.html
deleted file mode 100644
index 9a0f048..0000000
--- a/_includes/initialize_shuffle.html
+++ /dev/null
@@ -1,130 +0,0 @@
-<script type="text/javascript">
-$(document).ready(function() {
-    $('#toc').toc({ minimumHeaders: 0, listType: 'ul', showSpeed: 0, headers: 'h2,h3,h4' });
-});
-
-</script>
-<!-- shuffle -->
-<script>
-var shuffleme = (function( $ ) {
-  'use strict';
-
-  var $grid = $('#grid'),
-      $filterOptions = $('.filter-options'),
-      $sizer = $grid.find('.shuffle_sizer'),
-
-  init = function() {
-
-    // None of these need to be executed synchronously
-    setTimeout(function() {
-      listen();
-      setupFilters();
-    }, 100);
-
-    // instantiate the plugin
-    $grid.shuffle({
-      itemSelector: '[class*="col-"]',
-      sizer: $sizer    
-    });
-  },
-
-  // Set up button clicks
-  setupFilters = function() {
-    var $btns = $filterOptions.children();
-    $btns.on('click', function() {
-      var $this = $(this),
-          isActive = $this.hasClass( 'active' ),
-          group = isActive ? 'all' : $this.data('group');
-
-      // Hide current label, show current label in title
-      if ( !isActive ) {
-        $('.filter-options .active').removeClass('active');
-      }
-
-      $this.toggleClass('active');
-
-      // Filter elements
-      $grid.shuffle( 'shuffle', group );
-    });
-
-    $btns = null;
-  },
-
-  // Re layout shuffle when images load. This is only needed
-  // below 768 pixels because the .picture-item height is auto and therefore
-  // the height of the picture-item is dependent on the image
-  // I recommend using imagesloaded to determine when an image is loaded
-  // but that doesn't support IE7
-  listen = function() {
-    var debouncedLayout = $.throttle( 300, function() {
-      $grid.shuffle('update');
-    });
-
-    // Get all images inside shuffle
-    $grid.find('img').each(function() {
-      var proxyImage;
-
-      // Image already loaded
-      if ( this.complete && this.naturalWidth !== undefined ) {
-        return;
-      }
-
-      // If none of the checks above matched, simulate loading on detached element.
-      proxyImage = new Image();
-      $( proxyImage ).on('load', function() {
-        $(this).off('load');
-        debouncedLayout();
-      });
-
-      proxyImage.src = this.src;
-    });
-
-    // Because this method doesn't seem to be perfect.
-    setTimeout(function() {
-      debouncedLayout();
-    }, 500);
-  };      
-
-  return {
-    init: init
-  };
-}( jQuery ));
-
-
-
-$(document).ready(function() {
-  shuffleme.init();
-});
-
-    </script>
-
-    <!-- new attempt-->
-
-    <script>
-    $(document).ready(function() {
-     
-    /* initialize shuffle plugin */
-    var $grid = $('#grid');
-         
-    $grid.shuffle({
-        itemSelector: '.item' // the selector for the items in the grid
-    });
- 
-});</script>
-
-<script>
-$('#filter a').click(function (e) {
-    e.preventDefault();
-         
-    // set active class
-    $('#filter a').removeClass('active');
-    $(this).addClass('active');
-         
-    // get group name from clicked item
-    var groupName = $(this).attr('data-group');
-         
-    // reshuffle grid
-    $grid.shuffle('shuffle', groupName );
-});</script>
-
-
diff --git a/_includes/inline_image.html b/_includes/inline_image.html
deleted file mode 100644
index 1e7fd18..0000000
--- a/_includes/inline_image.html
+++ /dev/null
@@ -1 +0,0 @@
-<img class="inline" src="images/{{include.file}}" alt="{{include.alt}}" />
diff --git a/_includes/taglogic.html b/_includes/taglogic.html
deleted file mode 100644
index c2f76b0..0000000
--- a/_includes/taglogic.html
+++ /dev/null
@@ -1,32 +0,0 @@
-<p>The following pages and posts are tagged with <button type="button" style="cursor: default" class="btn btn-default navbar-btn">{{page.tagName}}</button></p>
-<table><thead><tr><th>Title</th><th>Type</th><th>Excerpt</th></tr></thead>
-    <tbody>
-    {% assign thisTag = page.tagName %}
-  {% for page in site.pages %}
-    {% for tag in page.tags %}
-        {% if tag == thisTag %}
-
-        <tr><td><a href="{{ page.url | remove: "/" }}">{{page.title}}</a></td>
-            <td><span class="label label-default">Page</span></td>
-          <td>{% if page.summary %} {{ page.summary | strip_html | strip_newlines | truncate: 160 }} {% else %} {{ page.content | truncatewords: 50 | strip_html }} {% endif %}</td>
-        </tr>
-        {% endif %}
-     {% endfor %}
-   {% endfor %}
-
-        {% assign thisTag = page.tagName %}
-        {% for post in site.posts %}
-        {% for tag in post.tags %}
-        {% if tag == thisTag %}
-
-        <tr><td><a href="{{ post.url | remove: "/" }}">{{post.title}}</a></td>
-            <td><span class="label label-primary">Post</span></td>
-            <td>{% if post.summary %} {{ post.summary | strip_html | strip_newlines | truncate: 160 }} {% else %} {{ post.content | truncatewords: 50 | strip_html }} {% endif %}</td>
-        </tr>
-        {% endif %}
-        {% endfor %}
-        {% endfor %}
-
-   </tbody>
-</table>
-
diff --git a/_includes/topnav.html b/_includes/topnav.html
index 236c9a1..f314c23 100644
--- a/_includes/topnav.html
+++ b/_includes/topnav.html
@@ -13,21 +13,21 @@
         <div id="navbar" class="navbar-collapse collapse">
             <ul class="nav navbar-nav navbar-right">
                 <!-- entries without drop-downs appear here -->
-                {% for entry in site.data.topnav.topnav %}
-                {% for item in entry.items %}
-                {% if item.external_url %}
+                {%- for entry in site.data.topnav.topnav %}
+                {%- for item in entry.items %}
+                {% if item.external_url -%}
                 <li><a href="{{item.external_url}}" target="_blank">{{item.title}}</a></li>
-                {% elsif page.url contains item.url %}
+                {%- elsif page.url contains item.url %}
                 <li class="active"><a href="{{ item.url | relative_url }}">{{item.title}}</a></li>
-                {% else %}
+                {%- else %}
                 <li><a href="{{ item.url | relative_url }}">{{item.title}}</a></li>
-                {% endif %}
-                {% endfor %}
-                {% endfor %}
+                {%- endif %}
+                {%- endfor %}
+                {%- endfor %}
                 <!-- entries with drop-downs appear here -->
                 <!-- conditional logic to control which topnav appears for the audience defined in the configuration file.-->
-                {% for entry in site.data.topnav.topnav_dropdowns %}
-                {% for folder in entry.folders %}
+                {% for entry in site.data.topnav.topnav_dropdowns -%}
+                {%- for folder in entry.folders %}
                 <li class="dropdown">
                     <a href="#" class="dropdown-toggle" data-toggle="dropdown">{{ folder.title }}<b class="caret"></b></a>
                     <ul class="dropdown-menu">
@@ -44,9 +44,6 @@
                 </li>
                 {% endfor %}
                 {% endfor %}
-                {% if site.feedback_disable == null or site.feedback_disable == false %}
-			{% include feedback.html %}
-		{% endif %}
 
                 <li>
                     <form id="search-demo-container" class="navbar-form navbar-right" role="search">
diff --git a/_layouts/default.html b/_layouts/default.html
index b371430..ff84838 100644
--- a/_layouts/default.html
+++ b/_layouts/default.html
@@ -38,18 +38,6 @@
       $('[data-toggle="tooltip"]').tooltip()
   })
   </script>
-  {% if page.datatable == true %}
-  <link rel="stylesheet" type="text/css" href="//cdn.datatables.net/1.10.5/css/jquery.dataTables.css">
-  <script type="text/javascript" charset="utf8" src="//cdn.datatables.net/1.10.5/js/jquery.dataTables.js"></script>
-  <script>
-  $(document).ready(function(){
-      $('table.datatable').DataTable( {
-          paging: false,
-          stateSave: true
-      });
-  });
-  </script>
-  {% endif %}
 </head>
 <body id="page-top">
 {% include topnav.html %}
@@ -60,9 +48,5 @@
 
 {% include footer.html %}
 
-{% if site.google_analytics %}
-{% include google_analytics.html %}
-{% endif %}
-
 </body>
 </html>
diff --git a/_layouts/default_deviceinfo.html b/_layouts/default_deviceinfo.html
index 59d78e1..01ecc29 100644
--- a/_layouts/default_deviceinfo.html
+++ b/_layouts/default_deviceinfo.html
@@ -39,18 +39,6 @@
             $('[data-toggle="tooltip"]').tooltip()
         })
     </script>
-    {% if page.datatable == true %}
-    <link rel="stylesheet" type="text/css" href="//cdn.datatables.net/1.10.5/css/jquery.dataTables.css">
-    <script type="text/javascript" charset="utf8" src="//cdn.datatables.net/1.10.5/js/jquery.dataTables.js"></script>
-    <script>
-        $(document).ready(function(){
-            $('table.datatable').DataTable({
-                paging: false,
-                stateSave: true
-            });
-        });
-    </script>
-    {% endif %}
 
 </head>
 <body id="page-top">
@@ -74,7 +62,4 @@
 {% include footer.html %}
 
 </body>
-{% if site.google_analytics %}
-{% include google_analytics.html %}
-{% endif %}
 </html>
diff --git a/_layouts/deviceinfo.html b/_layouts/deviceinfo.html
index 4dc0332..2edea59 100644
--- a/_layouts/deviceinfo.html
+++ b/_layouts/deviceinfo.html
@@ -6,28 +6,6 @@ layout: default_deviceinfo
    <h1 class="post-title-main">{{ page.title }}</h1>
 </div>
 
-{% if page.simple_map == true %}
-
-<script>
-    $(document).ready ( function(){
-        $('.box{{page.box_number}}').addClass('active');
-    });
-</script>
-
-{% include custom/{{page.map_name}}.html %}
-
-{% elsif page.complex_map == true %}
-
-<script>
-    $(document).ready ( function(){
-        $('.modalButton{{page.box_number}}').addClass('active');
-    });
-</script>
-
-{% include custom/{{page.map_name}}.html %}
-
-{% endif %}
-
 <div class="post-content">
 
    {% if page.summary %}
@@ -52,8 +30,4 @@ layout: default_deviceinfo
         {% endif %}
     </div>
 
-    {% if site.disqus %}
-{% include disqus.html %}
-    {% endif %}
-
 </div>
diff --git a/_layouts/page.html b/_layouts/page.html
index 8dd5214..004d6c0 100644
--- a/_layouts/page.html
+++ b/_layouts/page.html
@@ -6,57 +6,29 @@ layout: default
    <h1 class="post-title-main">{{ page.title }}</h1>
 </div>
 
-{% if page.simple_map == true %}
-
-<script>
-    $(document).ready ( function(){
-        $('.box{{page.box_number}}').addClass('active');
-    });
-</script>
-
-{% include custom/{{page.map_name}}.html %}
-
-{% elsif page.complex_map == true %}
-
-<script>
-    $(document).ready ( function(){
-        $('.modalButton{{page.box_number}}').addClass('active');
-    });
-</script>
-
-{% include custom/{{page.map_name}}.html %}
-
-{% endif %}
-
 <div class="post-content">
-
-   {% if page.summary %}
+    {% if page.summary -%}
     <div class="summary">{{page.summary}}</div>
-   {% endif %}
+    {%- endif %}
 
-    {% unless page.toc == false %}
-    {% include toc.html %}
-    {% endunless %}
+    {% unless page.toc == false -%}
+    {% include toc.html -%}
+    {%- endunless -%}
 
   {{content}}
 
     <div class="tags">
-        {% if page.tags != null %}
-        {% assign projectTags = site.data.tags.allowed-tags %}
-        {% for tag in page.tags %}
-        {% if projectTags contains tag %}
-        {% unless hasTags %}
-        {% assign hasTags = true %}
+        {% if page.tags != null and page.tags.size > 0 -%}
+        {% assign projectTags = site.data.tags.allowed-tags -%}
+        {% for tag in page.tags -%}
+        {% if projectTags contains tag -%}
+        {% unless hasTags -%}
+        {% assign hasTags = true -%}
         <b>Tags: </b>
-        {% endunless %}
+        {%- endunless %}
         <a href="{{ "tag_" | append: tag | append: ".html" }}" class="btn btn-default navbar-btn cursorNorm" role="button">{{page.tagName}}{{tag}}</a>
-        {% endif %}
-        {% endfor %}
-        {% endif %}
+        {%- endif %}
+        {%- endfor %}
+        {%- endif %}
     </div>
-
-    {% if site.disqus %}
-{% include disqus.html %}
-    {% endif %}
-
 </div>
diff --git a/_layouts/post.html b/_layouts/post.html
index 2fb7f90..0623361 100644
--- a/_layouts/post.html
+++ b/_layouts/post.html
@@ -27,8 +27,4 @@ layout: default
 
 </article>
 
-{% if site.disqus %}
-{% include disqus.html %}
-{% endif %}
-
 {% include footer.html %}
diff --git a/search.json b/search.json
index 833b71a..97bf4b3 100644
--- a/search.json
+++ b/search.json
@@ -6,35 +6,23 @@ search: exclude
 
 [
 
-{% for page in site.pages %}
-{% unless page.search == "exclude" or page.url contains "_info.html" or page.url contains "_install.html" or page.url contains "_build.html" or page.url contains "devices.html" or page.url contains ".css" %}
-{% unless page.url contains "/devices" and page.url contains "/build" or page.url contains "/devices" and page.url contains "/install" %}
+{%- for page in site.pages %}
+{%- unless page.search == "exclude" or page.url contains "_info.html" or page.url contains "_install.html" or page.url contains "_build.html" or page.url contains "devices.html" or page.url contains ".css" %}
+{%- unless page.url contains "/devices" and page.url contains "/build" or page.url contains "/devices" and page.url contains "/install" %}
 {
-{% if page.url contains "/devices" %}
-{% assign device = site.data.devices[page.device] %}
+{%- if page.url contains "/devices" %}
+{%- assign device = site.data.devices[page.device] %}
 "title": "{{ device.codename | escape }} - {{ device.vendor | escape }} {{ device.name }}",
-{% else %}
+{% else -%}
 "title": "{{ page.title | escape }}",
-{% endif %}
+{% endif -%}
 "tags": "{{ page.tags }}",
 "keywords": "{{page.keywords}}",
 "url": "{{ page.url | relative_url }}",
 "summary": "{{page.summary | strip }}"
 }
-{% if forloop.last == false %}, {% endif %}
-{% endunless %}
-{% endunless %}
-{% endfor %}
-
-{% for post in site.posts %}
-{
-"title": "{{ post.title | escape }}",
-"tags": "{{ post.tags }}",
-"keywords": "{{post.keywords}}",
-"url": "{{ post.url }}",
-"summary": "{{post.summary | strip }}"
-}
-{% unless forloop.last %},{% endunless %}
-{% endfor %}
-
+{%- if forloop.last == false %}, {% endif %}
+{%- endunless %}
+{%- endunless %}
+{%- endfor %}
 ]
diff --git a/sitemap.xml b/sitemap.xml
index 958799f..5d18502 100644
--- a/sitemap.xml
+++ b/sitemap.xml
@@ -5,22 +5,13 @@ search: exclude
 
 <?xml version="1.0" encoding="UTF-8"?>
 <urlset xmlns="http://www.sitemaps.org/schemas/sitemap/0.9">
-  {% for post in site.posts %}
-  {% unless post.search == "exclude" %}
-  <url>
-    <loc>{{site.url}}{{post.url}}</loc>
-  </url>
-  {% endunless %}
-  {% endfor %}
-
-
-  {% for page in site.pages %}
-  {% unless page.search == "exclude" %}
-  {% unless page.url contains "_info.html" or page.url contains "_install.html" or page.url contains "_build.html" or page.permalink == "devices.html" %}
+  {%- for page in site.pages %}
+  {%- unless page.search == "exclude" %}
+  {%- unless page.url contains "_info.html" or page.url contains "_install.html" or page.url contains "_build.html" or page.permalink == "devices.html" %}
   <url>
     <loc>{{site.url}}{{ page.url }}</loc>
   </url>
-  {% endunless %}
-  {% endunless %}
-  {% endfor %}
-</urlset>
\ No newline at end of file
+  {%- endunless %}
+  {%- endunless %}
+  {%- endfor %}
+</urlset>
-- 
2.17.1

